programmers/난이도별/level01.약수의_합/areum.js

```diff
@@ -0,0 +1,7 @@
+function solution(n) {
+    let answer = 0;
+    for(let i=1; i<=n; i++){
+        if(n%i==0) answer += i;
+    }
+    return answer;
```

Comment:
for(let i=1; i<=n/2; i++){
        if(n%i==0) answer += i;
    }
    return answer + n;